server/routes.py

```
@@ -20,3 +23,37 @@ async def chat_endpoint(request: ChatRequest) -> ChatResponse:
+                        lines = chunk.split("\n")
+                        s = (
+                            "event: delta\n"
+                            + "".join(f"data: {line}\n" for line in lines)
+                            + "\n"
+                        )
+                        yield s
+            except Exception as e:
+                print(f"Error during chat processing: {e}")
+                yield f"data: {str(e)}\n\n"
+            finally:
+                yield f"data: [DONE]"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)